File: Tarea 2/prueba.py
#Definir las variables 
A = 1
B = 10
C = 20
D = C + B
#mostrando variables
print("Esto es A:", A)
#Probando una forma para mostrar las variables 
print(f"D equivale a: {D}")
#Otra forma de mostrar variables 
print("mostrando D:", D)

F = D + B

print(F"F equivale a: {F}")
print("mostrando F:", F)

#Cambiando la variable 
A = 30 

print("Ahora A equivale a:", A)

#elevando una variable a una potencia "X"
A = 4
potencia = float(input("A que potencia quieres elevar A? "))

resultado = pow(A, potencia) 

#mostrando el resultado de la potencia 
print(f"A elevado a la potencia {potencia} es:", resultado)

#jugando con las variables
E = resultado * B

print("Mostrando E:",  E)

H = E / C

print("Mostrando H", H)

#convirtiendo los resultados a enteros
H = int(H)

print("Mostrando H:", H)

File: Tarea 2/prueba.cpp
#include <iostream>
#include <cmath>

using namespace std;

// Declarar las variables globales
int A = 1;
int B = 10;
int C = 20;

int main() {
    // Calcular D directamente
    int D = C + B;

    // Mostrar variables
    cout << "Esto es A: " << A << endl;
    cout << "D equivale a: " << D << endl;

    // Cambiando la variable A
    A = 30;
    cout << "Ahora A equivale a: " << A << endl;

    // Agregar más variables y operaciones
    int E = A * B;
    cout << "E equivale a: " << E << endl;

    int F = D * C;
    cout << "F equivale a: " << F << endl;

    // Elevando una variable a una potencia "X"
    int base = 4;
    double potencia;
    cout << "A que potencia quieres elevar A? ";
    cin >> potencia;

    // Calcular resultado directamente
    double resultado = pow(base, potencia);

    // Mostrar el resultado de la potencia
    cout << "A elevado a la potencia " << potencia << " es: " << resultado << endl;

    // Jugando con las variables
    double G = resultado + E;
    cout << "G equivale a: " << G << endl;

    double H = G / F;
    cout << "H equivale a: " << H << endl;

    // Convirtiendo los resultados a enteros
    int H_entero = static_cast<int>(H);
    cout << "Mostrando H como entero: " << H_entero << endl;

    return 0;
}
